src/entity.rs

```diff
@@ -1,7 +1,11 @@
 use ctype::isdigit;
-use entity_data;
 use std::char;
-use std::cmp::{min, Ordering};
+use std::cmp::min;
+
+use entities::ENTITIES;
+
+pub const ENTITY_MIN_LENGTH: usize = 2;
+pub const ENTITY_MAX_LENGTH: usize = 31;
```

Comment:
Ideally we'd calculate these from `ENTITIES` directly, otherwise we might get surprised one day if there's a new release of the `entities` crate and they no longer hold.